chapter_install/index_vn.md

```diff
@@ -136,7 +137,8 @@ through the first few chapters but you will want
 to access GPUs before running larger models.
 -->
 
-*dịch đoạn phía trên*
+Ngược lại, bạn có thể cài đặt phiên bản chỉ sử dụng CPU.
+Phiên bản này cũng thừa đủ để bạn có thể tiến hành các chương đầu tiên nhưng bạn sẽ phải sử dụng GPU để có thể chạy những mô hình lớn hơn.
 
 ```bash
 # For Windows users
```

Comment:
Những đoạn thế này sao mình không dịch luôn mà để lại mọi người nhỉ?

Reply:
comment trong code sửa lại cũng được nhưng có thể do bên bản gốc không yêu cầu dịch chăng

Reply:
@duythanhvn Theo anh không nên dịch code (trong đoạn ba dấu backtick). Kiến thức có thể dịch ra tiếng Việt nhưng khi viết code thì nên đặt tên biến và comment bằng tiếng Anh.

Reply:
> @duythanhvn Theo anh không nên dịch code (trong đoạn ba dấu backtick). Kiến thức có thể dịch ra tiếng Việt nhưng khi viết code thì nên đặt tên biến và comment bằng tiếng Anh.

Dạ, em chỉ đang nói đến mấy dòng như `# For Windows user`, `# For Linux and macOS users` thôi ạ. Hai dòng này dịch hay không thì cũng không khó hiểu, nhưng dịch ra thì có sự nhất quán hơn.